specs/partials.yml

```diff
@@ -106,6 +106,27 @@ tests:
        |
       /
 
+  - name: Standalone Indentation
+    desc: Indentation should only be applied after each newline in a partial.
```

Comment:
This description is not strictly true, because the indentation should also be applied to the first line, before any newline.